gremlin-checks/checks.groovy

```diff
@@ -195,3 +195,8 @@ check("route target belonging to several tenants (only RT matching the pattern '
      )
      .filter{it.get().value("display_name").matches(rtPattern)}
 )
+
+check("instance-ip without virtual-machine-interface",
+    g.V().hasLabel('instance_ip')
+         .not(__.out().hasLabel("virtual_machine_interface"))
+)
```

Comment:
Can you remove the "virtual-network with instance-ip but without any virtual-machine-interface" check as well?

Reply:
In checks.py too

Reply:
yes updated